backend/src/api/guilds/importExport.ts

```diff
@@ -35,7 +35,7 @@ const caseData = z.object({
   is_hidden: z.boolean(),
   pp_id: z.nullable(z.string()),
   pp_name: z.nullable(z.string()),
-
+  log_message_id: z.nullable(z.string()),
```

Comment:
Might need `.optional()` here to not break existing exports which don't include this field